test/datastore/syncstore.test.js

```diff
@@ -386,17 +386,8 @@ describe('SyncStore', function() {
 
     it('should call update() for an entity that contains an _id with special characters', function() {
       const store = new SyncStore(collection);
-      const specialSymbols = ['.', '$', '~', '>', '<', '!', '@'];
-      const savePromises = specialSymbols.map(symbol => {
-        const id = `${randomString()}${symbol}${randomString()}`;
-        return store.save({ _id: id })
-          .then(resp => {
-            expect(resp).toExist();
-            expect(resp._id).toEqual(id);
-          });
-      });
-
-      return Promise.all(savePromises);
+      const id = '.$~<>!@+_#';
+      return store.save({ _id: id });
```

Comment:
It was, but now I'm thinking we better make sure the id is indeed the same. At the time I thought successfully saving the item was enough, but it is better to make sure. I will add the check for the id.